src/connector/src/sink/kafka.rs

```diff
@@ -376,6 +379,9 @@ impl KafkaSinkWriter {
             schema,
             pk_indices,
             is_append_only,
+            future_delivery_buffer: Vec::new(),
+            // FIXME: Where to accept the input?
+            max_limit: Some(10),
```

Comment:
If it is currently a fixed value, we can put the fixed value in a global const named like `KAFKA_WRITER_MAX_QUEUE_SIZE`

Reply:
I'd prefer a global const at present, to keep it simple🤣.
Maybe integrate configurable buffer size in the future🤔️.